pandas/tests/extension/base/base.py

```diff
@@ -2,6 +2,9 @@
 
 
 class BaseExtensionTests:
+    # Whether the EA being tested supports __setitem__
+    _supports_setitem = True
```

Comment:
I guess this is ok, but in reality you should have a class which does this (future PR)